default.nix

```diff
@@ -39,6 +39,14 @@ rec {
       name = "ref";
       path = pkgs.writeText "${dockerImage.name}-image-tag" "${dockerImage.imageName}:${dockerImage.imageTag}";
     }
+    {
+      name = "image-repo";
+      path = pkgs.writeText "${dockerImage.name}-name" dockerImage.imageName;
+    }
+    {
+      name = "image-tag";
+      path = pkgs.writeText "${dockerImage.name}-name" dockerImage.imageTag;
```

Comment:
The filename could be suffixed with `-tag` instead of `-name`. This would improve the legibility of this nix file slightly.

Reply:
Good point, this was an oversight. Will fix.

Reply:
de76f931c1a3a44573722340da37ca62707c8e31